aiohttp/web_runner.py

```diff
@@ -97,7 +95,8 @@ def __init__(self, runner: 'BaseRunner',
     @property
     def name(self) -> str:
         scheme = 'https' if self._ssl_context else 'http'
-        return str(URL.build(scheme=scheme, host=self._host, port=self._port))
+        host = "0.0.0.0" if self._host is None else self._host
```

Comment:
I made it like this to be backwards compatible, but in most cases it's incorrect, as it can listen both on IPv4 and IPv6 (if available)

Reply:
Got you. Let's keep your version